### ISSUE TITLE
Error returned with redis logline

### ISSUE THREAD

**Issue by arpit1912** [DiceDB/dicedb-cli#7]
While running any wrong command, the CLI return back logline with redis in it. For example:

```
dicedb> SET k1 apple
OK
dicedb> GET k2
Error: redis: nil
dicedb> GET k1
apple
```

Expected behaviour:

It should have diceDB in the naming instead.

**Comment by Pratham-B-Parlecha** [comment]
Hey Arpit can you assign this issue to me

**Comment by roopeshsn** [comment]
I think this should be handled here: https://github.com/DiceDB/dicedb-go

**Comment by JyotinderSingh** [comment]
> I think this should be handled here: https://github.com/DiceDB/dicedb-go

That's correct. @Pratham-B-Parlecha Would you be interested in raising a fix for this issue on the dicedb-go repository? If not we can unassign this issue.

**Comment by Pratham-B-Parlecha** [comment]
@JyotinderSingh im interested in fixing this error on dicedb-go

**Comment by JyotinderSingh** [comment]
> @JyotinderSingh im interested in fixing this error on dicedb-go

Awesome, please create a bug for this on the repo. I'll assign it to you.

**Comment by JyotinderSingh** [comment]
> > @JyotinderSingh im interested in fixing this error on dicedb-go
> 
> Awesome, please create a bug for this on the sdk repo. I'll assign it to you.

**Comment by Pratham-B-Parlecha** [comment]
Hi @JyotinderSingh, could you please create the bug and assign it to me? I am unable to create the bug myself. 
Thank you!

**Comment by JyotinderSingh** [comment]
> Hi @JyotinderSingh, could you please create the bug and assign it to me? I am unable to create the bug myself. 
> 
> Thank you!

Please start working on the fix in your fork and raise a PR. I'll try to figure out the permission issue in the meantime.
